### PR TITLE
Improve theme toggle

### PR DESCRIPTION
**Pull request by Aloso**
Replaces the sheep in the theme toggle with a sun and improves HTML markup.

### CODE REVIEW

File: src/components/ThemeToggle.tsx [hunk 1]
```diff
@@ -12,13 +13,21 @@ const ThemeToggle = () => {
...
-      <img
-        src={theme === "light" ? halfMoon : sheep}
-        alt={"Moon Icon for Theme Toggle"}
-        width={"50"}
-      />
-    </div>
+    <button className={styles.button} onClick={handleChangeTheme}>
+      {theme == "light" ? (
+        <img
+          src={halfMoon}
```
Comment (quintus-luis) [inline src/components/ThemeToggle.tsx:19]: Why you would prefer two image tags over one?

Comment (Aloso) [reply src/components/ThemeToggle.tsx:19]: It seemed more readable to me. The alternative is

```js
      <img
        src={theme == "light" ? halfMoon : sun}
        alt={
          theme == "light"
            ? "Moon Icon for Theme Toggle"
            : "Sun Icon for Theme Toggle"
        }
        title={
          theme == "light"
            ? "Zum dunklen Design wechseln"
            : "Zum hellen Design wechseln"
        }
      />
```

Comment (LeoTuet) [reply src/components/ThemeToggle.tsx:19]: I agree with Aloso - two tags are more readable.

Comment (94726) [reply src/components/ThemeToggle.tsx:19]: You could also have two functions return the corresponding values similar to [leos new PR](https://github.com/LeoTuet/ui-we-want-to-sleep/pull/70/commits/0d57d89792eb3fd919961c830bb00cb8f00c46e9#diff-49b1b26f2b78139b3a19a58f388ed5ad7b91e9d4eeb736e19402c8daf55051d5R31)

Comment (Aloso) [reply src/components/ThemeToggle.tsx]: I changed it now.

**Comment by Aloso** [comment]
@quintus-luis Is this good to merge?

**Comment by quintus-luis** [comment]
> @quintus-luis Is this good to merge?

Yes!

**Comment by 94726** [comment]
might fix #79 in here

**Comment by Aloso** [comment]
@94726 I'd rather do that in a separate PR.

**Comment by quintus-luis** [comment]
> @94726 I'd rather do that in a separate PR.

We should think about whether the toggle is placed correctly at this point anyway. In my opinion, the selection of the language is much more important at our school and the website adapts to the browser theme anyway.

**Comment by Aloso** [comment]
@quintus-luis I think the website should also adapt to the browser's language settings automatically. I didn't check if it does; if not we should open an issue for that.

**Comment by quintus-luis** [comment]
it does. So need for an issue

**Pull request merged**